seccomp/README.md

```diff
@@ -128,49 +292,48 @@ Profiles can contain more granular filters based on the value of the arguments t
     * SCMP_CMP_EQ - equal to
     * SCMP_CMP_GE - greater than
     * SCMP_CMP_GT - greater or equal to
-    * SCMP_CMP_MASKED_EQ - masked equal: true iff `(value & arg == valueTwo)`
+    * SCMP_CMP_MASKED_EQ - masked equal: true if `(value & arg == valueTwo)`
 * `value` is a parameter for the operation
 * `valueTwo` is used only for SCMP_CMP_MASKED_EQ
 
-The rule matches if **all** args match. To achieve the effect of an or, add multiple rules.
+The rule only matches if **all** args match. Add multiple rules to achieve the effect of an OR.   << **NOTE TO LAB OWNER: IS THAT STATEMENT CORRECT?**
 
-Strace can be used to get a list of all system calls made by a program.
+`Strace` can be used to get a list of all system calls made by a program.
 It's a very good starting point for writing seccomp policies.
 Here's an example of how we can list all system calls made by `ls`:
 
 ```
 $ strace -c -f -S name ls 2>&1 1>/dev/null | tail -n +3 | head -n -2 | awk '{print $(NF)}'
+access
+arch_prctl
+brk
+close
+execve
+<SNIP>
+statfs
+write
 ```
 
-### Demo
+The output above shows all of the syscalls that will need enabling for a container running the `ls` program to work.
```

Comment:
"all of the syscalls that will need"
1. missing a word
2. we should make this statement weaker because you'll also need the minimum necessary syscalls to start a container (the ones runc calls between when it applies the profile and when the user's program is exec'd)